contributors/design-proposals/pod-priority-resourcequota.md

```diff
@@ -55,27 +55,22 @@ type ResourceQuotaSpec struct {
   // If not specified, this ResourceQuota applies to all priority classes.
   // +optional
   PriorityClassName string
-  // The priority value. Various system components use this field to find the
-  // priority of the ResourceQuota. When Priority Admission Controller is enabled, it
-  // prevents users from setting this field. The admission controller populates
-  // this field from PriorityClassName.
-  // +optional
-  Priority *int32
 }
 ```
+PriorityClassName in ResourceQuota is matched against PriorityClassName in the Pod, including the case where PriorityClassName is not set. In detail:
 
-* One could expect when `PriorityClassName` field is set, it indicates that this quota applies to Pod with same priority class. 
-* If this field is not present, this quota applies to all priority classes.
+* One could expect when `PriorityClassName` field is set, it indicates that this quota applies to to pods with the same priority class.
+* If this field is not present, it indicates this quota matches to pods that do not have a PriorityClassName (Pods' integer priorities would be ignored).
 
-It is worth noting that quota is assigned per namespace, not per user, in Kubernetes. Adding priority to ResourceQuota will not change anything with respect to the scope of ResourceQuota. In other words, when priority is defined for a ResourceQuota object, the quota is applied to all pods at that priority in the specified namespace.
+It is worth noting that quota is assigned per namespace, not per user, in Kubernetes. Adding priority class name to ResourceQuota will not change anything with respect to the scope of ResourceQuota. In other words, when priority class name is defined for a ResourceQuota object, the quota is applied to all pods at that priority class in the specified namespace.
 
-Just like pod spec, we also have `Priority *int32` in `ResourceQuta` spec. So the class don't need to be resolved every time it is being used.
+Unlike pod spec, we do not have `Priority *int32` in `ResourceQuta` spec.  It's because we allow ResourceQuota with empty PriorityClassName and such a ResourceQuota matches pods that do not have a PriorityClassName. We think this design is more straightforward for end user.
```

Comment:
The "It's because..." part is bit confusing. The fact that resource quota with empty priority class name applies to pods with no priority class name is not the reason for not using an integer value of  priority. I would rephrase this paragraph as:

Unlike pod spec, `ResourceQuota` does not have `Priority *int32` in its spec.  
As explained above, `ResourceQuota` with a PriorityClassName matches against pods with the same PriorityClassName, regarless of the integer value of priority.